### PR TITLE
description that ChargeItem and PayItem sums should be equal

### PR DESCRIPTION
**Pull request by Bluette-C-Riviere**

### CODE REVIEW

File: doc/general/data-structures/data-structures.md [hunk 1]
```diff
@@ -185,6 +188,8 @@ namespace fiskaltrust.ifPOS.v0
 
 Payment entries are used for receipt requests as well as for receipt responses.
 
+The sum of `ftPayItems` and the sum of `ftChargeItems`should be equal except for very specific cases. If that is not the case a warning will be returned.
```
Comment (AxelKutschera) [inline doc/general/data-structures/data-structures.md:191]:
```suggestion
In every request using the ftReceiptCase 0x????000000000001, the sum of the filed `amount` of all `ftPayItems` and the sum of the field `amount` of all `ftChargeItems`should be equal. If that is not the case a warning will be returned except for very specific cases.
```

**Review by AxelKutschera — COMMENTED** [review]
Please revise my suggestion, if its not correct in detail.

File: doc/general/data-structures/data-structures.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ namespace fiskaltrust.ifPOS.v0
 
 Payment entries are used for receipt requests as well as for receipt responses.
 
-The sum of `ftPayItems` and the sum of `ftChargeItems`should be equal except for very specific cases. If that is not the case a warning will be returned.
+In every request using the ftReceiptCase 0x????000000000001, the sum of the filed `amount` of all `ftPayItems` and the sum of the field `amount` of all `ftChargeItems`should be equal. If that is not the case a warning will be returned except for very specific cases.
```
Comment (AxelKutschera) [inline doc/general/data-structures/data-structures.md:191]:
```suggestion
In every request using the ftReceiptCase 0x????000000000001, the sum of the field `amount` of all `ftPayItems` and the sum of the field `amount` of all `ftChargeItems`should be equal. If that is not the case a warning will be returned except for very specific cases.
```

**Review by AxelKutschera — COMMENTED** [review]
1 Typo

**Review by TSchmiedlechner — DISMISSED** [review]

**Comment by Bluette-C-Riviere** [comment]
dead links updated to functionnal links

**Review by TSchmiedlechner — APPROVED** [review]

**Comment by TSchmiedlechner** [comment]
LGTM 👍🏻

**Pull request merged**